documentation/docs/guides/creating-plans.md

```diff
@@ -0,0 +1,304 @@
+---
+sidebar_position: 17
+title: Creating Plans with the /plan prompt
+sidebar_label: Creating Plans
+---
+
+Starting a project without a clear plan is like building a house without a blueprint. It can lead to:
+
+* Confusion about what to do
+* Wasted time and effort
+* Projects that grow too big
+
+A good plan keeps everyone on track and helps measure progress. That's why the Goose CLI includes the `/plan` prompt completion command to help break down your projects into clear, manageable steps.
+ 
+:::tip Plans in the Goose desktop
```

Comment:
```suggestion
:::tip Plans in the Goose Desktop
```

Reply:
for consistency in our docs..let's write Goose Desktop -- capitalized..but this is a good read thus far.